experiments/ClimaEarth/test/amip_test.jl

```diff
@@ -0,0 +1,11 @@
+## AMIP testing
+# This script runs a coarse AMIP simulation for a short period of time.
+# This is meant to be used by upstream packages to test if changes to those
+# packages affect the interfaces used by ClimaCoupler.jl.
+# Note that this test doesn't check stability or correctness of the model.
+
+driver_path = joinpath("..", "run_amip.jl")
+config_file = "amip_test.yml"
+
+# Run the AMIP test
+run(`julia --project $driver_path --config_file=$config_file --job_id="amip_test"`)
```

Comment:
This will behave differently depending on the folder where it is run from (because of `driver_path`).

I think we should be able to `include("run_amip.jl")` instead

Reply:
Yeah, I wasn't sure about the paths. I didn't want to just call `include` because that won't let us specify the config file. Also `run_amip` is in a different directory so I didn't think we're able to include it directly like that

Reply:
You can specify the config file by setting the `ARGS` variable, and you can also include across directories. I think `include("../run_amip.jl`)` should work (at the end of the day, this is how we include stuff in `src`).